xchange-independentreserve/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>com.xeiam.xchange</groupId>
+		<artifactId>xchange-parent</artifactId>
+		<version>3.0.1-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>xchange-independentreserve</artifactId>
+
+	<name>XChange Independent Reserve</name>
+	<description>XChange implementation for the Independent Reserve</description>
+
+	<url>http://xeiam.com</url>
+	<inceptionYear>2012</inceptionYear>
+
+	<organization>
+		<name>Xeiam, LLC</name>
+		<url>http://xeiam.com</url>
+	</organization>
+
+	<!-- Parent provides default configuration for dependencies -->
+	<dependencies>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>2.7</version>
```

Comment:
Can you avoid jodatime? XChange strives to keep the dependencies to a minimum as has so far avoided jodatime.